.github/workflows/pull-request.yml

```
@@ -466,3 +466,54 @@
         run: |
           echo "Error: Pull request target branch must be 'develop-postgres'. Please refer PR_GUIDELINES.md"
           exit 1
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)